go/mysql/flavor_mariadb.go

```diff
@@ -154,6 +155,25 @@ func (mariadbFlavor) status(c *Conn) (SlaveStatus, error) {
 	if err != nil {
 		return SlaveStatus{}, vterrors.Wrapf(err, "SlaveStatus can't parse MariaDB GTID (Gtid_Slave_Pos: %#v)", resultMap["Gtid_Slave_Pos"])
 	}
+	filePos, err := strconv.Atoi(resultMap["Exec_Master_Log_Pos"])
+	if err != nil {
+		return SlaveStatus{}, fmt.Errorf("invalid FilePos GTID (%v): expecting pos to be an integer", resultMap["Exec_Master_Log_Pos"])
+	}
+
+	status.FilePosition.GTIDSet = filePosGTID{
+		file: resultMap["Relay_Master_Log_File"],
+		pos:  filePos,
+	}
+
+	fileRelayPos, err := strconv.Atoi(resultMap["Relay_Log_Pos"])
```

Comment:
This should be `Read_Master_Log_Pos`: the position within the master's binlog file. `Relay_Log_Pos` is the position within each replica's local relay log file, which is not comparable against the value from any other replica.

Reply:
Done.